app/styles/crisp.less

```diff
@@ -180,3 +180,37 @@ footer {
     }
   }
 }
```

Comment:
I think we must use css variables for colors, widths, heights, etc. In case if we will have an intention to make redesign later.